Remove amdgpu installation from quick start (#168)
ROCm/rocm-install-on-linux: docs/tutorial/quick-start.rst

.. _rocm-install-quick:

*************************************************************

> ROCm quick start installation guide for Linux

Reviewer: Change again:
Quick start installation guide

Author: https://github.com/ROCm/rocm-install-on-linux/pull/176

*************************************************************

This topic provides basic installation instructions for ROCm on Linux using your distribution’s native package manager. Review your required installation instructions by selecting your operating system and version, and then run the provided commands in your terminal.

> For more in-depth installation instructions, refer to :ref:`rocm-install-overview`.

Reviewer: Need a new paragraph here for Docker - will send over teams once I have it.

Author: To be added in https://github.com/ROCm/rocm-install-on-linux/pull/176

> ROCm quick start install guide for Linux

Reviewer: ```suggestion
ROCm quick start installation guide for Linux
```

*************************************************************

> The following are steps to install ROCm on Linux via your distribution's native package manager. To get
> started with ROCm on Linux, select your operating system and version, then run the provided commands in your
> terminal.

Reviewer: ```suggestion
This topic provides basic installation instructions for ROCm on Linux using your distribution’s native package manager. Review your required installation instructions by selecting your operating system and version, and then run the provided commands in your terminal.
```

                {% for (os_version, os_release) in config.html_context['ubuntu_version_numbers'] %}
                .. tab-item:: {{ os_version }}

                    .. code-block:: bash
                        :substitutions:

                        sudo apt update
                        sudo apt install "linux-headers-$(uname -r)" "linux-modules-extra-$(uname -r)"

>                         # Add the current user to the render and video groups (see prerequisites).

Reviewer 2: I initially had this comment on a separate line.  Feedback was to put it on the same line.  Let's see if others give the same feedback again :)

Reviewer 3: Do we really need to have the comment at all?

Reviewer 2: we added it from Saad's feedback, if you recall.

Reviewer 3: Right.  I forgot where that came from.  Then maybe we should leave it as a single line.

                    .. code-block:: bash
                        :substitutions:

                        sudo apt update
                        sudo apt install "linux-headers-$(uname -r)" "linux-modules-extra-$(uname -r)"
                        sudo usermod -a -G render,video $LOGNAME # Adding current user to Video, Render groups. See prerequisites.
                        # Add the current user to the render and video groups (see prerequisites).
                        sudo usermod -a -G render,video $LOGNAME
                        wget https://repo.radeon.com/amdgpu-install/|amdgpu_version|/ubuntu/{{ os_release }}/amdgpu-install_|amdgpu_install_version|_all.deb
                        sudo apt install ./amdgpu-install_|amdgpu_install_version|_all.deb

>                         sudo apt update

Reviewer 3: The "sudo apt update" line needs to stay.  Do not remove it.

Author: @dbieleck We're making `sudo apt update` the first step. Is that fine with you?

```shell
sudo apt update
sudo apt install "linux-headers-$(uname -r)" "linux-modules-extra-$(uname -r)"
# Add the current user to the render and video groups (see prerequisites).
sudo usermod -a -G render,video $LOGNAME
wget https://repo.radeon.com/amdgpu-install/6.1.2/ubuntu/jammy/amdgpu-install_6.1.60102-1_all.deb
sudo apt install ./amdgpu-install_6.1.60102-1_all.deb
sudo apt install amdgpu-dkms rocm
```

Reviewer 3: No.  You need to have the update after the install of the amdgpu-install script, or the instructions will fail:

...
sudo apt install ./amdgpu-install_6.1.60102-1_all.deb
sudo apt update
sudo apt install amdgpu-dkms rocm

The apt update on the top line can remain.

Author: @dbieleck Ohh ok, thanks. Fixed.

                .. tab-item:: {{ os_version }}

                    .. code-block:: bash
                        :substitutions:

                {% if os_version == "15.4" %}
                        # Installing Perl module from SLES 15.5, as it was removed from 15.4
                        sudo zypper addrepo https://download.opensuse.org/repositories/devel:/languages:/perl/15.5/devel:languages:perl.repo
                {% else %}
                        sudo zypper addrepo https://download.opensuse.org/repositories/devel:languages:perl/{{ os_version}}/devel:languages:perl.repo
                {% endif %}
                        sudo zypper install kernel-default-devel
                        sudo usermod -a -G render,video $LOGNAME # Add the current user to the render and video groups
                        sudo zypper --no-gpg-checks install https://repo.radeon.com/amdgpu-install/|amdgpu_version|/sle/{{ os_version }}/amdgpu-install-|amdgpu_install_version|.noarch.rpm
                        sudo zypper refresh
                        sudo zypper install amdgpu-dkms rocm
                {% endfor %}

.. note::

   To apply all settings, reboot your system.

> If you have issues with your installation, see :doc:`Troubleshooting <../how-to/native-install/install-faq>`.

Reviewer: New paragraph to add at bottom - link to the post install page.

After completing the installation, review the **post-installation instructions**.

Author: Added in https://github.com/ROCm/rocm-install-on-linux/pull/176

                .. tab-item:: {{ os_version }}

                    .. code-block:: bash
                        :substitutions:

                {% if os_version == "15.4" %}
                        # Installing Perl module from SLES 15.5, as it was removed from 15.4
                        sudo zypper addrepo https://download.opensuse.org/repositories/devel:/languages:/perl/15.5/devel:languages:perl.repo
                {% else %}
                        sudo zypper addrepo https://download.opensuse.org/repositories/devel:languages:perl/{{ os_version}}/devel:languages:perl.repo
                {% endif %}
                        sudo zypper install kernel-default-devel
                        sudo usermod -a -G render,video $LOGNAME # Add the current user to the render and video groups
                        sudo zypper --no-gpg-checks install https://repo.radeon.com/amdgpu-install/|amdgpu_version|/sle/{{ os_version }}/amdgpu-install-|amdgpu_install_version|.noarch.rpm
                        sudo zypper refresh
                        sudo zypper install amdgpu-dkms rocm
                {% endfor %}

> .. important::

Reviewer: Let's use a Note here - the Google standard is Tips/Note/Warning/Caution

>    Reboot your system for all settings to take effect.

Reviewer: ```suggestion
To apply all settings, reboot your system.
```

> If you encounter issues with your installation, refer to :doc:`Troubleshooting <../how-to/native-install/install-faq>`.

Reviewer: ```suggestion
If you have issues with your installation, see :doc:`Troubleshooting <../how-to/native-install/install-faq>`.
```
